packages/server/lib/util/ci_provider.js

```diff
@@ -262,6 +262,10 @@ const _providerCiParams = () => {
       'GITHUB_RUN_ID',
       'GITHUB_RUN_ATTEMPT',
       'GITHUB_REPOSITORY',
+      'GITHUB_BASE_REF',
+      'GITHUB_HEAD_REF',
+      'GITHUB_REF_NAME',
+      'GITHUB_REF',
```

Comment:
While I was in here, I added the GHA variables that may be helpful in the future to the ci_params mapping. These additions aren't essential to correct the current fallback values, but they may be useful in the future for further inspection.